test/suites/build/fixtures/cloud_build/Kubefile

```diff
@@ -1,10 +1,10 @@
-FROM sealer-io/test:v1
+FROM sealer-io/kubernetes:v1.19.8
```

Comment:
cloud build will push image automatically ,this action will update our base image ,so i suggested that we use the test base image.